utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -103,83 +105,160 @@ int main(int argc, char **argv)
+            if (layout_switch == 1)
+                printf("\nUsage: ./Tensor_host_pln3 <src1 folder> <src2 folder (place same as src1 folder for single image functionalities)> <u8 = 0 / f16 = 1 / f32 = 2 / u8->f16 = 3 / u8->f32 = 4 / i8 = 5 / u8->i8 = 6> <outputFormatToggle (pkd->pkd = 0 / pkd->pln = 1)> <case number = 0:86> <verbosity = 0/1>\n");
+            if (layout_switch == 2)
+            {
+                printf("\nUsage: ./Tensor_host_pln1 <src1 folder> <src2 folder (place same as src1 folder for single image functionalities)> <u8 = 0 / f16 = 1 / f32 = 2 / u8->f16 = 3 / u8->f32 = 4 / i8 = 5 / u8->i8 = 6> <outputFormatToggle (pkd->pkd = 0 / pkd->pln = 1)> <case number = 0:86> <verbosity = 0/1>\n");
+                if (atoi(argv[5]) != 0)
+                {
+                    printf("\nPLN1 cases don't have outputFormatToggle! Please input outputFormatToggle = 0\n");
+                    return -1;
+                }
+            }
+        }
+        else
+        {
 
-    // Set case names
+            if (layout_switch == 0)
+                printf("\nUsage: ./Tensor_host_pkd3 <src1 folder> <src2 folder (place same as src1 folder for single image functionalities)> <dst folder> <u8 = 0 / f16 = 1 / f32 = 2 / u8->f16 = 3 / u8->f32 = 4 / i8 = 5 / u8->i8 = 6> <outputFormatToggle (pkd->pkd = 0 / pkd->pln = 1)> <case number = 0:86> <number of iterations > 0> <verbosity = 0/1>>\n");
+            if (layout_switch == 1)
+                printf("\nUsage: ./Tensor_host_pln3 <src1 folder> <src2 folder (place same as src1 folder for single image functionalities)> <dst folder> <u8 = 0 / f16 = 1 / f32 = 2 / u8->f16 = 3 / u8->f32 = 4 / i8 = 5 / u8->i8 = 6> <outputFormatToggle (pkd->pkd = 0 / pkd->pln = 1)> <case number = 0:86> <number of iterations > 0> <verbosity = 0/1>>\n");
+            if (layout_switch == 2)
+                printf("\nUsage: ./Tensor_host_pln1 <src1 folder> <src2 folder (place same as src1 folder for single image functionalities)> <dst folder> <u8 = 0 / f16 = 1 / f32 = 2 / u8->f16 = 3 / u8->f32 = 4 / i8 = 5 / u8->i8 = 6> <outputFormatToggle (pkd->pkd = 0 / pkd->pln = 1)> <case number = 0:86> <number of iterations > 0> <verbosity = 0/1>>\n");
+            {
+                if (atoi(argv[5]) != 0)
+                {
+                    printf("\nPLN1 cases don't have outputFormatToggle! Please input outputFormatToggle = 0\n");
+                    return -1;
+                }
+            }
+        }
+    }
+    int ip_channel;
+    if (layout_switch == 0 || layout_switch == 1)
+    {
+        ip_channel = 3;
+    }
+    else
+    {
+        ip_channel = 1;
+    }
 
-    char funcType[1000] = {"Tensor_HOST_PKD3"};
+    // Set case names
+    string funcType;
+    if (layout_switch == 0)
```

Comment:
There are multiple places where we are rechecking layout_switch value and updating variables accordingly. Some of the instances are
Line [176](https://github.com/sampath1117/rpp/blob/037ad043c6f3909f890c8a1edabef3b9c10028b8/utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp#L176)
Line [187](https://github.com/sampath1117/rpp/blob/037ad043c6f3909f890c8a1edabef3b9c10028b8/utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp#L187) 
Line [273](https://github.com/sampath1117/rpp/blob/037ad043c6f3909f890c8a1edabef3b9c10028b8/utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp#L273)

Declare all the variables at once and merge them in single if-else instead of multiple if-else